bash_nc_listen_sample.json

```diff
@@ -12,10 +12,10 @@
   "dependencies": [],
   "env": {},
   "executor": "",
-  "health_checks": [],
+  "health_checks": [{"gracePeriodSeconds":300,"intervalSeconds":30,"timeoutSeconds":10,"maxConsecutiveFailures":3,"portIndex":0,"protocol":"TCP"}],
   "labels": {},
   "max_launch_delay_seconds": 3600,
-  "ports": [31108],
```

Comment:
Why remove the port number here?

Reply:
marathon is choosing the port for us, we don't need to list anything here.